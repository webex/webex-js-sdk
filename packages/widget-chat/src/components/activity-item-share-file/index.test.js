import React from 'react';
import renderer from 'react-test-renderer';

import ActivityItemShareFile from '.';

describe(`ActivityItemShareFile component`, () => {
  const props = {
    file: {
      displayName: `testImage.js`,
      url: `http://cisco.com`
    },
    onDownloadClick: jest.fn()
  };

  const component = renderer.create(
    <ActivityItemShareFile {...props} />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
