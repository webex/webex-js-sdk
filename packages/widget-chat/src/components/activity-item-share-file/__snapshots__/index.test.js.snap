exports[`ActivityItemShareFile component renders properly 1`] = `
<div
  className="activity-share-item">
  <div
    className="share-file-icon">
    <div
      className="icon"
      title={undefined} />
  </div>
  <div
    className="share-meta">
    <div
      className="share-file-info">
      <div
        className="share-name">
        testImage.js
      </div>
      <div
        className="share-file-props">
        <span
          className="share-file-size">
          0 Bytes
        </span>
        <span
          className="share-file-type" />
      </div>
    </div>
    <div
      className="share-item-actions">
      <div
        className="share-action-item">
        <button
          className="icon-button"
          onClick={[Function handleDownloadClick]}
          onKeyUp={[Function handleDownloadClick]}>
          <div
            className="icon"
            title="Download this file" />
        </button>
      </div>
    </div>
  </div>
</div>
`;
